src/extension.ts

```diff
@@ -78,7 +78,7 @@ function surroundWithTry(selection: vscode.Selection) {
         `${prefix}try {`,
         ...indentLines(lines),
         `${prefix}} catch (error) {`,
-        `${prefix}${indent}`,
+        `${prefix}${indent}console.error(error);`,
```

Comment:
Note: it could be even more useful to make sure the entire error object is printed, with something like this:

```js
console.log(require('util').inspect(error, { depth: null }));
```